src/Client.php

```diff
@@ -531,4 +531,17 @@ public function searchAssets(array $params) {
     return $results;
   }
 
+  public function downloadAsset(int $assetID) {
```

Comment:
As much as I would love to have the int typehint, we can't use that because we have to support PHP 5.6 here for the time being.